testbed/testbed/test_case.go

```diff
@@ -315,7 +315,7 @@ func (tc *TestCase) WaitForN(cond func() bool, duration time.Duration, errMsg ..
 
 // WaitFor is like WaitForN but with a fixed duration of 10 seconds
 func (tc *TestCase) WaitFor(cond func() bool, errMsg ...interface{}) bool {
-	return tc.WaitForN(cond, time.Second*10, errMsg...)
+	return tc.WaitForN(cond, time.Second*12, errMsg...)
```

Comment:
Any particular reason for this change? Did we have a test that was on the fence with 10 sec?

Reply:
I added to more time to see if it resolved flaky test issue. I reverted back now.